commit cf4182b2424159df07e6466cb1e437f091e0e99a
Author: Prabha Murugesan <prabha.murugesan@morsemicro.com>
Date:   Mon Mar 3 18:27:46 2025 +1100

    morse-regdb: (APP-4159, PR#ToDo) Revert S1G channel changes done for SG region

diff --git a/artefacts/repo_channels.csv b/artefacts/repo_channels.csv
index 41f156d..2917dc8 100644
--- a/artefacts/repo_channels.csv
+++ b/artefacts/repo_channels.csv
@@ -491,21 +491,14 @@
 SG,1,9,17,66,867.5,2.77,2.77,Singapore,29.14,False,100.0,0.0,1000.0,1
 SG,1,11,17,66,868.5,2.77,2.77,Singapore,29.14,False,100.0,0.0,1000.0,1
 SG,2,10,19,67,868.0,2.77,2.77,Singapore,29.14,False,100.0,0.0,1000.0,1
-SG,1,31,18,68,917.5,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
-SG,1,33,18,68,918.5,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
-SG,1,35,18,68,919.5,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
 SG,1,37,18,68,920.5,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
 SG,1,39,18,68,921.5,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
 SG,1,41,18,68,922.5,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
 SG,1,43,18,68,923.5,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
 SG,1,45,18,68,924.5,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
-SG,2,32,20,61,918.0,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
-SG,2,36,20,61,920.0,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
-SG,2,40,20,61,922.0,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
-SG,2,44,20,61,924.0,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
-SG,4,34,21,62,919.0,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
-SG,4,42,21,62,923.0,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
-SG,8,38,31,63,921.0,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
+SG,2,38,20,69,921.0,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
+SG,2,42,20,69,923.0,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
+SG,4,40,21,70,922.0,100.0,100.0,Singapore,22.15,False,0.0,0.0,0.0,1
 SI,1,1,6,66,863.5,100.0,100.0,Slovenia,16.13,False,0.0,0.0,0.0,1
 SI,1,3,6,66,864.5,100.0,100.0,Slovenia,16.13,False,0.0,0.0,0.0,1
 SI,1,5,6,66,865.5,100.0,100.0,Slovenia,16.13,False,0.0,0.0,0.0,1
